icubam/predicu/plot/barplot_inputs_per_day.py

```diff
@@ -1,7 +1,7 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 
-data_source = ["icubam"]
+data_source = ["bedcounts"]
```

Comment:
`icubam` should contain the same data (but possibly with different pre-processing and bedcounts)